### ISSUE TITLE
how to enable accessibility features of Electron

### ISSUE THREAD

**Issue by vojtapolasek** [GitSquared/edex-ui#826]
Hi,
I like your project. I am a blind Linux user. Applications using Electron have recently become accessible. But I think the application has to enable the accessibility at compliation time. How can I enable it? Any modification to build configuration?

**Comment by GitSquared** [comment]
Hi! Thanks for your interest.
Edex is a bit old and the Electron version it is built with does not have many accessibility features, i'm afraid... but maybe i can figure something out for you.
Would you mind reaching me at gabriel@saillard.dev or eventually on Twitter dms (@ GitSquared)?
